src/tests/test_fd_util.h

```diff
@@ -33,6 +33,17 @@ class AlgorithmTest : public ::testing::Test {
         };
     }
 
+    static algos::StdParamsMap MaxLHSParamMap(tests::TableConfig const& config,
+                                              unsigned int max_lhs_) {
+        using namespace config::names;
+        return {
+                {kTable, config.MakeInputTable()},
+                {kError, config::ErrorType{0.0}},
+                {kSeed, decltype(algos::pyro::Parameters::seed){0}},
+                {kMaximumLhs, max_lhs_},
+        };
+    }
```

Comment:
I think we don't need a separate function for this, better add parameter with the default value to the existing `GetParamMap`